### ISSUE TITLE
Bug: Not receiving forwarded port number

### ISSUE THREAD

**Issue by jpcapone** [qdm12/gluetun#2435]
### Is this urgent?

None

### Host OS

Docker

### CPU arch

None

### VPN service provider

Custom

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2024-08-23T13:50:02.262Z (commit ff7cadb)

### What's the problem 🤔

Not receiving forwarded port number - using custom type but the provider is ProtonVPN

### Share your logs (at least 10 lines)

```plain text
2024-08-23T22:36:55Z INFO [wireguard] Wireguard setup is complete. Note Wireguard is a silent protocol and it may or may not work, without giving any error message. Typically i/o timeout errors indicate the Wireguard connection is not working.
2024-08-23T22:36:55Z INFO [healthcheck] healthy!
2024-08-23T22:36:55Z INFO [dns] downloading hostnames and IP block lists
2024-08-23T22:36:55Z INFO [dns] DNS server listening on [::]:53
2024-08-23T22:36:56Z INFO [dns] ready
2024-08-23T22:36:56Z INFO [ip getter] Public IP address is 89.187.180.34 (United States, Illinois, Chicago)
2024-08-23T22:36:56Z INFO [vpn] You are running on the bleeding edge of latest!
2024-08-23T22:36:56Z INFO [port forwarding] starting
2024-08-23T22:36:56Z ERROR [vpn] starting port forwarding service: port forwarding for the first time: getting external IPv4 address: executing remote procedure call: gateway IP is unspecified
```

### Share your configuration

```yml
services:
  gluetun:
    image: qmcgaw/gluetun
    cap_add:
      - NET_ADMIN
    environment:
      - VPN_SERVICE_PROVIDER=custom
      - VPN_TYPE=wireguard
      - VPN_ENDPOINT_IP=XXXXXXXXX
      - VPN_ENDPOINT_PORT=51820
      - WIREGUARD_PUBLIC_KEY=XXXXXXXX
      - WIREGUARD_PRIVATE_KEY=sssssssssssssssss
      - WIREGUARD_ADDRESSES=10.2.0.2/32
      - VPN_PORT_FORWARDING=on
      - VPN_PORT_FORWARDING_PROVIDER=protonvpn
      - SHADOWSOCKS=on
      - SHADOWSOCKS_PASSWORD=sssssssssssssssssssss
      - HTTP_CONTROL_SERVER_ADDRESS=:8000
```

**Comment by CherubUltima** [comment]
I got the same error, until this is fixed I reverted back to v3.38.0, port forwarding working there

**Comment by DARTE5** [comment]
Same error. I think he might have accidentally pushed to latest. It seems like 3.39 is supposed to be the latest release. I switched to qmcgaw/gluetun:v3.39 and it’s working again.

**Comment by jpcapone** [comment]
reverting did the trick.  Thanks for the assist!

**Comment by qdm12** [comment]
This is due to the upgrade to netlink 2 days ago 540acc915d56ba1c0a2c51edd2f0f1399a9e861a thanks for the fast feedback guys 💪 💯 

Working on a fix...

**Comment by qdm12** [comment]
Fixed in 01fa9934bce4fe49e8fe7c97ecfd74eaf2c9d807 ! Thanks again for taking the time to report this and comment 😉 !
